### ISSUE TITLE
The padding compute for Keras & Tensorflow are invalid

### ISSUE THREAD

**Issue by lygstate** [mil-tokyo/webdnn#660]
The convert result are 3, but the expected are 2.

**Comment by Kiikurage** [comment]
Can you give me more information, how to reproduce?

**Comment by lygstate** [comment]
@Kiikurage Yeap, wait a moment 

```
0 input_1 (InputLayer)             (None, 96, 96, 1)     0
____________________________________________________________________________________________________
conv2d_1 (Conv2D)                (None, 48, 48, 64)    3200        input_1[0][0]
```


```
{"name": "conv2d_1", "trainable": true, "filters": 64, "kernel_size": [7, 7], "strides": [2, 2], "padding": "same", "data_format": "channels_last", "dilation_rate": [1, 1], "activation": "linear", "use_bias": true, "kernel_initializer": {"class_name": "VarianceScaling", "config": {"scale": 2.0, "mode": "fan_in", "distribution": "normal", "seed": null}}, "bias_initializer": {"class_name": "Zeros", "config": {}}, "kernel_regularizer": {"class_name": "L1L2", "config": {"l1": 0.0, "l2": 9.999999747378752e-05}}, "bias_regularizer": null, "activity_regularizer": null, "kernel_constraint": null, "bias_constraint": null}
```

For this layer, Keras and the generated backend generating different result.
The padding in Keras & Tensorflow are 2, and the padding in WebDNN backend are 3. So that WebDNN produced invalid result.

**Comment by Kiikurage** [comment]
Staled. Maybe this problem is fixed in #695.
